docs/workflow/testing/libraries/filtering-tests.md

```diff
@@ -176,3 +234,67 @@ dotnet build <csproj_file> /t:Test /p:TargetOS=windows
 ```sh
 dotnet build <csproj_file> /t:Test /p:TargetOS=OSX /p:WithCategories="OuterLoop;failing""
 ```
+
+## SkipOnCoreClrAttribute
+This attribute is used to disable a test under specific conditions, only when run with CoreCLR (it doesn't affect tests run with Mono). Typically, this is when there is a failure in a particular test run configuration, such as under GCStress or JitStress.
```

Comment:
I believe that `SkipOn...` attributes are meant to be used for permanent disable, without an ActiveIssue.

Reply:
ideally, but there are some filters possible on them that aren't possible on ActiveIssue.